src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureBase {
+    private int firstSide;
+    private int secondSide;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int firstSide, int secondSide, int height) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstSide + secondSide) / 2 * height;
```

Comment:
All magic numbers in your code should be constants. The number `2` used in the area calculation could be declared as a constant with a meaningful name.